docs/styling.md

```diff
@@ -36,26 +36,26 @@ Our current styling architecture is still quite nascent and is very likely to un
-| `--media-range-track-width`         | `width`            | `100%`                    | width of the underlying slider's track display                                                                             | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
-| `--media-range-track-border`        | `border`           | `none`                    | border of the underlying slider's track display                                                                            | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
-| `--media-range-track-border-radius` | `border-radius`    | `0`                       | border radius of the underlying slider's track display                                                                     | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
-| `--media-range-track-background`    | `background`       | `#eee`                    | background color of the underlying slider's track display                                                                  | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
-| `--media-range-track-box-shadow`    | `box-shadow`       | `none`                    | box shadow of the underlying slider's track display                                                                        | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
-| `--media-range-track-transition`    | `transition`       | `none`                    | apply a [transition](https://developer.mozilla.org/en-US/docs/Web/CSS/transition) to the underlying slider's track display | Applied via `::-webkit-slider-runnable-track` and similar [pseudo-element selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-slider-runnable-track) |
+| Name                                   | CSS Property       | Default Value             | Description                                                                                                                | Notes                                                                                                                                                                  |
```

Comment:
Oh man, this diff is not helpful. 😆 I'm assuming there's nothing critical happening here though.